css/style.css

```
@@ -16,4 +24,101 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+#nav-bar {position:fixed;
+          top:0; 
+          height: 70px;
+          width:100%;
+          right:40px;
+          background-color: white;
+                   }
 
+    #nav-bar ul {list-style-type:none; 
+                display:flex; 
+                justify-content: flex-end;
+                }     
+  
+    #nav-bar ul a:hover {color:#F15B2A;
+                        }
```

```suggestion
#nav-bar ul a:focus,
#nav-bar ul a:hover {
  color:#F15B2A;
}
```
We should also add `:focus` so that keyboard users who use the `Tab` key to jump to the link also get a change in colour.

Also, for coding the formatting of the code matters in software teams. Typically, it's structured like in my suggestion. This applies to the whole file